vrk-kpa/suomidigi
# SUOPA-838 Header theme

To test: 
- `make fresh drush-cr`
- Open mobile navigation proto: https://xd.adobe.com/view/1067743e-f959-428d-664c-974d183bc243-6656/screen/36dc28e9-3fef-425e-b3c8-e9584df3290e/Suomidigi-etusivu-navi-mobile/
- Open desktop proto: https://xd.adobe.com/view/d3c60a1d-1d76-4a8b-4797-030e9c2e717f-a5f7/
- Open https://suomidigi.docker.sh in desktop mode (do not log in)
   - Check that the header is similar to proto
   - Test the active elements on header (search, language selection and log in button)
   - Log in to the site and check the header again.
   - Test that the "First name - last name" button functions properly
   - Log out
- Switch to mobile mode
   - Repeat the same steps as in desktop mode
